src/main/webapp/resources/js/pages/user/components/UserChangePasswordForm.jsx

```diff
@@ -42,16 +46,20 @@ export function UserChangePasswordForm({ userId, requireOldPassword }) {
   };
 
   return (
-    <>
+    <Skeleton loading={isLoading}>
       <Typography.Title level={5}>
         {i18n("UserChangePasswordForm.title")}
       </Typography.Title>
+      <Typography.Text type="secondary" hidden={!userDetails.domainAccount}>
```

Comment:
**chore (blocking):** Instead of just hiding this, can you wrap it in an if statement so that it never gets rendered to the page.

Reply:
Also on the `UserSecurityPage` add a condition to the if statement on line 25 since we do not need to see the password policy if the user is LDAP

Reply:
Changes added in https://github.com/phac-nml/irida/pull/1481/commits/93f0836065f94f1f59d46f14dcdaad55fc8b2258

Reply:
On the admin > list > account page, thank you for removing the alert with the password requirements, I can still see the reset password stuff though, can you please also remove that.

Reply:
Oh I missed that because I didn't have email configured. Should be fixed now. [f49faa5](https://github.com/phac-nml/irida/pull/1481/commits/f49faa549e1edb21051a40c3c1cb6c2bfc071d1a)